ui/raidboss/data/README.txt

```diff
@@ -1,150 +0,0 @@
-  id: 'id string'
-  // Regular expression to match against.
-  regex: /trigger-regex-(with-position-1)-here/,
-  // Example of a locale-based regular expression for the 'fr' locale.  If Options.Language == 'fr', then
-  // regexFr (if it exists) takes precedence over regex.  Otherwise, it is ignored.  This is only an
-  // example for french, but other locales behave the same, e.g. regexEn, regexKo.
-  regexFr: /trigger-regex-(with-position-1)-here-but-in-french/,
-  // Time to wait before showing it once the regex is seen. May be a number or a function(data, matches) that returns a number.
-  delaySeconds: 0,
-  // Time to wait in seconds before showing this trigger again.  May be a number of a function(data, matches).  The time to wait begins at the regex (and not after the delaySeconds).  These triggers will not run anything.  This can be used for catching the first of many events if there's log spam.
-  suppressSeconds: 0,
-  // Number of seconds to show the trigger for. May be a number or a function(data, matches) that returns a number.
-  durationSeconds: 3,
-  // Text to show with info importance. May be a string or a function(data, matches) that returns a string.
-  infoText = 'Info',
-  // Text to show with alert importance. May be a string or a function(data, matches) that returns a string.
-  alertText = 'Info',
-  // Text to show with alarm importance. May be a string or a function(data, matches) that returns a string.
-  alarmText = 'Info',
-  // Sound file to play, or one of 'Info', 'Alert', 'Alarm', or 'Long'. Paths to sound files are
-  // relative to the ui/raidboss/ directory.
-  sound: '',
-  // Volume between 0 and 1 to play |sound|.
-  soundVolume: 1,
-  // A function(data, matches) to test if the trigger should fire or not. If it does not return
-  // true, nothing is shown/sounded/run. This is the first thing that is run on the trigger.
-  condition: function(data, matches) { return true if it should run }
-  // A function(data, matches) to run. This runs as the first step after condition.
-  preRun: function(data, matches) { do stuff.. },
-  // A function(data, matches) to run. This runs as the last step when the trigger fires.
-  run: function(data, matches) { do stuff.. },
-  // If this is true, the trigger is completely disabled and ignored.
-  disabled: true,
-},
-
-Any field that can return a function (e.g. infoText, alertText, alarmText, tts) can also return a localized
-object, e.g. instead of returning 'Get Out', they can return {en: 'Get Out', fr: 'something french'}
-instead.  Fields can also return a function that return a localized object as well.  If the current locale
-does not exist in the object, the 'en' result will be returned.
-
-
-The full order of evaluation of functions in a trigger is:
-1. condition
-2. preRun
-3. delaySeconds
-4. durationSeconds
-5. suppressSeconds
-6. infoText
-7. alertText
-8. alarmText
-9. tts
-10. run
-
-
-Timeline Readme
```

Comment:
Would you mind including this bit too?

Reply:
Oof. I was working off a local copy of readme.txt from before you redid the file structure. Sorry I missed including that.